android/app/build.gradle

```diff
@@ -305,7 +305,8 @@ android {
             keyPassword 'android'
         }
 
-        def pass = getPassword("rainbow","rainbow-android-key")
+
+        def pass = System.getenv('RAINBOW_KEY_ANDROID_PASSWORD')
```

Comment:
And how this will work locally? I don't think we should have in our global envs? Can we support both mechanisms